definitions/ext-sap_system/definition.yml

```diff
@@ -0,0 +1,30 @@
+domain: EXT
+type: SAP_SYSTEM
+goldenTags:
+  - instrumentation.provider
+  - sap.funtional_area
+  - sap.system
+
+synthesis:
+  rules:
+  # Telemetry from New Relic SAP integration
+  - identifier: SAP_EID
+    name: SYS_ID
+    encodeIdentifierInGUID: true
+    conditions:
+    - attribute: SAP_ETYPE
+      value: SYSTEM
+    tags:
+      sap.functional_area:
+      SYS_ID:
```

Comment:
this is the name of the entity, why it needs to be a tag?

Sorry i just noticed this now

Reply:
The intention for adding SYS_ID attribute as the sap.system is to use it for all applicable entity types that belong to a system, including the system itself; then use the tag to create a workload that group different type of entities of a sap systems. Then this will allow us to present a view from SAP System organized. If there is a way of taking advantage of this attribute being the name of the sap_system entity itself, we'd like to know and consider to use it.

Reply:
with this you are looking at a query like: `tags.sap_system = 'system name'` without this you would need to do `tags.sap_system = 'system name' and name = 'system name'`

and probably to be safe `tags.sap_system = 'system name' and (name = 'system name' and type = 'SAP_SYSTEM' )`

I don't love the idea of having tags of the entity name but i can see the use case you are looking for. I'll let it go for now unless it becomes a problem in the future.